holoviews/plotting/bokeh/util.py

```diff
@@ -40,6 +40,11 @@
 # and can therefore be safely ignored.
 IGNORED_MODELS = ['LinearAxis']
 
+# Model priority order to ensure some types are updated before others
+MODEL_PRIORITY = ['Range1d', 'Title', 'Image', 'LinearColorMapper',
+                  'Plot', 'Range1d', 'LinearAxis', 'ColumnDataSource']
```

Comment:
Yes, although I've not so far encountered any issues apart from the color mapper one, but we may well run into others at some point.